localnet/poktrolld/config/supplier1_stake_config.yaml

```diff
@@ -1,3 +1,27 @@
+# This file is used to stake a Supplier or update an existing Supplier's information in the Pocket Network.
```

Comment:
Can create a `supplier_stake_config_example.yaml` and copy-paste this over with all the comments/details?

No one will ever read a config that's used for business logic.

If you're up to it, doing this for the other staking configs will go a long way as well.